src/io/indexed_recordio_split.h

```diff
@@ -32,6 +33,7 @@ class IndexedRecordIOSplitter : public InputSplitBase {
                           const int seed = 0) {
     this->shuffle_ = shuffle;
     if (shuffle) SetRandomSeed(seed);
+    this->shuffle_group_size_ = dmlc::GetEnv("DMLC_INDEXED_RECORDIO_SHUFFLE_GROUP_SIZE", 16);
```

Comment:
Can you add some explanation to docs/faq/env_var.md, people won't know it otherwise

Reply:
It is in dmlc-core so I think it should start with DMLC_, no? I will add the explanation once I make PR to MXNet with updated dmlc-core.

Reply:
lol, my bad. didn't notice that